websites/O/Origin/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+    clientId: "760611329588985886"
+  }),
+  presenceData: PresenceData = {
+    largeImageKey: "icon"
+  },
+  customData = false;
+
+presence.on("UpdateData", async () => {
+
+  if (document.location.pathname == "/my-home") {
+    presenceData.details = "Browsing the homepage";
+  } else if (document.location.pathname.startsWith("/store/download")) {
+    presenceData.details = "Installing the Origin launcher";
+    };
+    if (document.location.pathname.startsWith("/store")) {
+    presenceData.details = "Browsing for a game";

```

Comment:
you should also make it like changing, when you go to a games store page, it shows the name of the game etc.

Reply:
ok

Reply:
done...